### PR TITLE
Postgres collector public integration 

### PR DESCRIPTION
**Pull request by yotamloe**
* Ability to add custom queries
* Ability to collect metrics from multiple instances
* Ability to add custom labels per instance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,106 +56,43 @@ otel:
...
+      pg_password: pass
+      pg_labels:
+        - alias: rds-2
 ```
 Mount the configuration file to your container:
 ```shell
 docker run --rm --name logzio/postgres-rds-collector \
 -v <<path_to_config_file>>:/config_files/config.yml \
-logzio/postgres-rds-collector
+logzio/mobileye-rds-collector
```
Comment (Doron-Bargo) [inline README.md:93]: why its called mobileye?

File: README.md [hunk 2]
```diff
@@ -257,5 +128,5 @@ docker run --rm --name logzio/postgres-rds-collector \
 -p 55679:55679 \
 -p 13133:13133 \
 -p 1777:1777 \
-logzio/postgres-rds-collector
+logzio/mobileye-rds-collector
```
Comment (Doron-Bargo) [inline README.md:131]: why its called mobileye?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
looking good - please look on my commnets

File: README.md [hunk 3]
```diff
@@ -40,65 +32,17 @@ is "us-east-1", and the slug for Canada (Central) is "ca-central-1".
...
--e LOGZIO_LOG_TOKEN=<<LOGZIO_LOG_TOKEN>> \
-logzio/postgres-rds-collector
-```
-
 #### Full list of configurable environment variables
 
 | Environment variable | Description |
 |---|---|
-| AWS_REGION (Required) | Your region's slug. You can find this in the AWS Console region menu (in the top menu, to the right).  **Note:** This is the region that you will collect metrics from. |
 | LOGZIO_REGION (Required)| Your Logz.io region code. For example if your region is US, then your region code is `us`. You can find your region code here: https://docs.logz.io/user-guide/accounts/account-region.html#regions-and-urls. |
```
[the remark this answers is not in the document]
Comment (yotamloe) [reply README.md:39]: We have a [function](https://github.com/logzio/postgres-rds-collector/blob/mobileye/config.py#L50-L62) that generates the listener URL based on this input, that's was the approach in previous integrations like cloudwatch-metrics. I can easily remove this if you think that the correct approach

File: Dockerfile [hunk 4]
```diff
@@ -9,45 +9,13 @@ COPY requirements.txt requirements.txt
...
 COPY queries queries
-COPY cw_namespaces cw_namespaces
 COPY testdata testdata
 COPY builder.py builder.py
 COPY config.py config.py
 COPY input_validator.py input_validator.py
-COPY cloudwatch_exporter-0.11.0-jar-with-dependencies.jar cloudwatch_exporter-0.11.0-jar-with-dependencies.jar
 
 # Download opentelemetry binary
 COPY otelcontribcol_linux_amd64 otelcontribcol_linux_amd64
```
[the remark this answers is not in the document]
Comment (yotamloe) [reply Dockerfile]: Done

**Review by Doron-Bargo — APPROVED** [review]
lgtm

**Pull request merged**